desktop/apps/buy_art/stylesheets/buttons.styl

```diff
@@ -0,0 +1,288 @@
+// @require '../../stylus_lib'
+// @require '../../../../node_modules/artsy-ezel-components/spinners'
+// @require '../../follow_button'
+
+.circle-button
```

Comment:
When you say existing stylesheet do you mean copy the style rules that we are using from `buttons.styl` into `index.styl`? We actually don't need the `buttons.styl` in its entirety. We are thinking it may be best just to use what we need.